_includes/navigation.html

```diff
@@ -10,7 +10,7 @@
       <li class="dropdown">
         <a href="#" class="dropdown-toggle" data-toggle="dropdown" >Documentation<i class="caret"></i></a>
         <ul class="dropdown-menu">
-            {% for version in site.versions reversed %}
+            {% for version in site.versions.rotate(-1) reversed %}
```

Comment:
If https://github.com/Katello/katello.org/pull/248/files#diff-def425679cdefbdbe06edfb202e71137R87 were to prepend instead of append, would that prevent the need to require this?